### ISSUE TITLE
sample/fibo/fibo got segmentation fault with CXXFLAGS='-march=native -O2'

### ISSUE THREAD

**Issue by YutaroOrikasa** [YutaroOrikasa/user-thread#1]
```
mkdir build_
cd build_
CXXFLAGS='-march=native -O2' cmake .. && make
./sample/fibo/fibo
```

got segmentation fault.
But `CXXFLAGS='-march=native -O3` didn't.

**Comment by YutaroOrikasa** [comment]
After adding debug output toggle feature, `CXXFLAGS='-march=native -O2' cmake -DORKS_USERTHREAD_DEBUG_OUTBUT=ON` got segfault but without `-DORKS_USERTHREAD_DEBUG_OUTBUT=ON` does not.

**Comment by YutaroOrikasa** [comment]
This issue had been fixed by commit b52652b67aeee314c449ee492f783d5b7cf51453.
